src/etc/dec2flt_table.py

```diff
@@ -105,6 +105,7 @@ def error(f, e, z):
 
 //! Tables of approximations of powers of ten.
 //! DO NOT MODIFY: Generated by `src/etc/dec2flt_table.py`
+#![cfg_attr(feature = "cargo-clippy", allow(unreadable_literal))]
```

Comment:
Can annotations like this be omitted? We don't run clippy in the main repository regularly so I'd prefer to not land annotations like this which are basically unused by all our infrastructure.

Reply:
Will do!